specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2383 @@
+        }
+      ]
+    },
+    "AnalyzeConversationJobStatistics": {
+      "description": "Contains the statistics for the job submitted.",
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/ConversationRequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "ConversationTasksState": {
+      "description": "Contains the state for the tasks being executed as part of the analyze conversation job submitted.",
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "description": "Count of tasks completed successfully.",
+              "type": "integer"
+            },
+            "failed": {
+              "description": "Count of tasks that failed.",
+              "type": "integer"
+            },
+            "inProgress": {
+              "description": "Count of tasks in progress currently.",
+              "type": "integer"
+            },
+            "total": {
+              "description": "Total count of tasks submitted as part of the job.",
+              "type": "integer"
+            },
+            "items": {
+              "description": "List of results from tasks (if available).",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalyzeConversationJobResult"
+              }
+            }
+          },
+          "required": [
+            "total",
+            "completed",
+            "failed",
+            "inProgress"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+        "tasks"
+      ],
+      "type": "object"
+    },
+    "AnalyzeConversationJobResult": {
+      "type": "object",
+      "description": "Container for results for all tasks in the conversation job.",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationResultsKind"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskState"
+        },
+        {
+          "$ref": "common.json#/definitions/TaskIdentifier"
+        }
+      ],
+      "required": [
+        "kind"
+      ]
+    },
+    "ConversationResultBase": {
+      "type": "object",
+      "description": "Shared attributes for all conversational task results.",
+      "required": [
+        "id",
+        "warnings"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique, non-empty conversation identifier."
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/ConversationWarning"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/ConversationStatistics"
+        }
+      }
+    },
+    "ConversationError": {
```

Comment:
Do we need a separate object for errors for this API?  The schema looks similar to what we already have used for TA, with the only difference being in the description.  Could we generalize the description of one error object to be used across multiple APIs?

Reply:
We can update both DocumentError and ConversationError to InputError, but that will be a backward incompatible change. Will that be OK?

Reply:
The REST API review board would prefer that all errors use the standard error object found in common.json. `additionalProperties` is `true` so you can add to it - typically in `details` - whatever you want.

Reply:
We can merge it in the next version.